userguide/snippets/alertevents.rst

```diff
@@ -53,10 +53,11 @@ must be dismissed by the user.
    +-------------------+------------------------+
 
 
-Close an alert message by hovering over it until
-:guilabel:`Click to Dismiss` appears. There is also an option to
-:guilabel:`CLEAR ALL ALERTS`. Close all messages to remove any
-notification badge from the alerts icon.
+Close an alert message by clicking
+:guilabel:`Dismiss`. There is also an option to
+:guilabel:`Dismiss All Alerts`. Close all messages to remove any
```

Comment:
Needs to be consistent in terms.  The button is Dismiss, but then the message says "Close".
```suggestion
:guilabel:`Dismiss All Alerts`. Dismissing all alerts removes the
```